### PR TITLE
dist: build_and_test.sh: don't hard-code base branch

### PR DESCRIPTION
**Pull request by kaspar030**
This PR should fix murdock for non-master branches.

### CODE REVIEW

**Comment by miri64** [comment]
ACK when murdock agrees (and fingers crossed it works for other branches then master ;-))

File: dist/tools/travis-scripts/build_and_test.sh [hunk 1]
```diff
@@ -7,6 +7,8 @@
 # directory for more details.
 #
 
+CI_BASE_BRANCH=${CI_BASE_BRANCH:-master}
```
Comment (miri64) [inline dist/tools/travis-scripts/build_and_test.sh]: optional, that's why I only say it after ACK: wouldn't an `export` make sense here?

Comment (kaspar030) [inline dist/tools/travis-scripts/build_and_test.sh]: It looked to me like all tools take the base branch as parameter.
(Within murdock, that variable is exported into the environment anyways.)

Comment (miri64) [inline dist/tools/travis-scripts/build_and_test.sh]: That's why I said optional. I'm thinking more about future scripts here ;-)

**Comment by kaspar030** [comment]
Build output looks fine to me! &go.

File: dist/tools/travis-scripts/build_and_test.sh (hunk 1 above)
Comment (miri64) [inline dist/tools/travis-scripts/build_and_test.sh]: But if you want to merge, please do so ;-)

**Pull request merged**